src/platform/datapath_raw.h

```diff
@@ -256,8 +256,13 @@ typedef struct CXPLAT_SOCKET {
     void* CallbackContext;
     QUIC_ADDR LocalAddress;
     QUIC_ADDR RemoteAddress;
-    BOOLEAN Wildcard;   // Using a wildcard local address. Optimization to avoid always reading LocalAddress.
-    BOOLEAN Connected;  // Bound to a remote address
+    BOOLEAN Wildcard;           // Using a wildcard local address. Optimization
+                                // to avoid always reading LocalAddress.
+    BOOLEAN Connected;          // Bound to a remote address
+    uint8_t CibirIdLength;      // Value of 0 indicates CIBIR isn't used
+    uint8_t CibirIdOffsetSrc;
```

Comment:
>CibirIdOffsetSrc [](http://example.com/codeflow?start=12&length=16)

Src and Dst mean CIBIR ID in source CID and destination CID?

Reply:
Yes, the offsets in the source and destination CIDs.

Reply:
maybe worth adding a line comment for them. Not immediately clear for me especially when CIBIR support for core is done in another PR.

Reply:
What would you like me to add? I personally think it's mostly self-explanatory. You still figured it out in the end without a comment. 😄

Reply:
"// CIBIR ID offsets in source CID and destination CID"? Up to you. I figured it out after opening the PR for core and the draft you wrote :smile:. Maybe it's just me not knowledgeable enough in QUIC specs.

Reply:
I improved the comments. Let me know what you think.

Reply:
Thanks! Much clearer.